includes/class-wc-calypso-bridge-customize-store.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Adds Customize Store task related functionalities
+ *
+ * @package WC_Calypso_Bridge/Classes
+ * @since   1.0.0
+ * @version 2.2.14
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC Calypso Bridge Customize Store
+ */
+class WC_Calypso_Bridge_Customize_Store {
+
+	/**
+	 * Class instance.
+	 *
+	 * @var WC_Calypso_Bridge_Customize_Store instance
+	 */
+	protected static $instance = false;
+
+	/**
+	 * Get class instance
+	 */
+	public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * Constructor.
+	 */
+	private function __construct() {
+
+		// Only in Ecommerce or Free Trial
+		if ( ! wc_calypso_bridge_has_ecommerce_features() && ! wc_calypso_bridge_is_ecommerce_trial_plan() ) {
+			return;
+		}
+
+		add_action( 'current_screen', array( $this, 'mark_customize_store_task_as_completed_on_site_editor' ) );
```

Comment:
```suggestion
		add_action( 'load-site-editor.php', array( $this, 'mark_customize_store_task_as_completed_on_site_editor' ) );
```

You can use the [load-{pagenow} hook](https://developer.wordpress.org/reference/hooks/load-pagenow/) to attach directly to the editor screen and avoid the need for the `get_current_screen()` check in `mark_customize_store_task_as_completed_on_site_editor()`. Hat tip to @ilyasfoo for showing me this recently!

Reply:
TIL! Thank you for the tip 👍